packages/react-combobox/etc/react-combobox.api.md

```diff
@@ -0,0 +1,203 @@
+## API Report File for "@fluentui/react-combobox"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import type { ComponentProps } from '@fluentui/react-utilities';
+import type { ComponentState } from '@fluentui/react-utilities';
+import type { ForwardRefComponent } from '@fluentui/react-utilities';
+import type { PositioningShorthand } from '@fluentui/react-positioning';
+import * as React_2 from 'react';
+import type { Slot } from '@fluentui/react-utilities';
+
+// @public
+export const Combobox: ForwardRefComponent<ComboboxProps>;
```

Comment:
Nit: this may have been discussed before but should this be called `ComboBox` with a capital B "Box"?

Reply:
I... don't know if that particular question was discussed. I don't have a strong opinion either way 😅.

I did open this on Open UI which is semi-related, but doesn't entirely answer which we should use: https://github.com/openui/open-ui/pull/425

Reply:
I don't really have a strong opinion either :) It just struck me as odd that the "b" in box wouldn't be capitalized when we were talking about this component earlier but the lowercase "b" is definitely the norm in your research.

Reply:
you know what, let's capitalize it. It's ComboBox in v8, and I see no reason to change it :)